ampersand-select-view.js

```diff
@@ -42,6 +42,8 @@ function SelectView (opts) {
     this.startingValue = opts.value;
     this.yieldModel = (opts.yieldModel === false) ? false : true;
 
+    this.beforeSubmit = opts.beforeSubmit;
```

Comment:
This may need to be something like:

`if(opts.beforeSubmit) this.beforeSubmit = opts.beforeSubmit`